docs/reference/motlp.md

```diff
@@ -173,11 +175,34 @@ When creating a Kubernetes secret, always encode the full string in Base64, incl
 
 ::::::
 
+% ## Rate limiting and throttling
+% 
+% The following rate limits apply to the {{motlp}}:
+% 
+% * n requests per second per API key
+% * m requests per minute per API key
+% 
+% If you exceed these limits, you will receive an HTTP 429 status code.
+
+## Data retention
```

Comment:
Can we add a section, calling out which data-stream the data is routed to based on the feedback here - https://github.com/elastic/ingest-dev/issues/5625#issuecomment-3169509361

we can take a tough summary from this - https://github.com/elastic/opentelemetry-dev/blob/main/docs/design-decisions/ingest/mapping.md#otel-specific-data-streams

Reply:
Added in https://github.com/elastic/opentelemetry/pull/398/commits/191846eda61ae1019000278a7ca3c83a5076045e